src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public void setColour(String colour) {
```

Comment:
The method name should be `setColor` instead of `setColour` to match the variable name.